eis_toolkit/spatial_analyses/distance_computation.py

```diff
@@ -0,0 +1,81 @@
+import geopandas as gpd
+import numpy as np
+from rasterio import profiles, transform
+from shapely.geometry import Point
+
+from eis_toolkit import exceptions
+
+
+def _smallest_distance_to_point(point: Point, geometries: gpd.GeoDataFrame) -> float:
+    """Calculate smalled distance from geometries to point."""
+    # This could be sped up by using a spatial index to first choose only
+    # geometries that are close to the point
+    distances_to_point = geometries.distance(point)
+    smallest_distance = distances_to_point.min()
```

Comment:
To fix no-any-return linter error:
```suggestion
    shortest_distance: float = distances_to_point.min()
```

Reply:
Fixed!